.github/workflows/nightwatch-build.yaml

```diff
@@ -97,7 +95,7 @@ jobs:
           project_name: ${{ env.project }}
           environment: ${{ inputs.environment }}
           gke_auth: true
-          gke_cluster: ${{ inputs.gke_cluster }}
+          gke_cluster: ${{ inputs.gke_cluster || inputs.environment }}
```

Comment:
Sure why not, aber sei dir bewusst, das hält jetzt noch 4-8 Wochen, und ab dann wird immer `staging != main-staging-yy-nn` sein -- aber man kann dann natürlich bis auf weiteres das letztere hier als Default setzen.